flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/functions/InternalWindowFunction.java

```diff
@@ -46,5 +46,5 @@
 	 *
 	 * @throws Exception The function may throw exceptions to fail the program and trigger recovery.
 	 */
-	public abstract void apply(KEY key, W window, IN input, Collector<OUT> out) throws Exception;
+	public abstract void process(KEY key, W window, IN input, Collector<OUT> out) throws Exception;
```

Comment:
why is the internal API different from the user facing `ProcessWindowFunction` ? How can users get window information through this API ?